common/src/main/java/com/arcbees/beeshop/common/api/ApiPaths.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2015 ArcBees Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.arcbees.beeshop.common.api;
+
+public class ApiPaths {
+    public static final String ROOT = "/api";
+
+    public static final String PAYMENT = "/payment";
+    public static final String STRIPE = "/stripe";
```

Comment:
I extracted a commit from another branch to make this PR, so none for now :) Stripe will be a subpath of payment ("/payment/stripe").